src/server/req_rescq.c

```diff
@@ -352,7 +354,7 @@ req_confirmresv(struct batch_request *preq)
 				str_time = ctime(&(presv->ri_wattr[RESV_ATR_retry].at_val.at_long));
 				if (str_time != NULL) {
 					str_time[strlen(str_time)-1] = '\0';
-					(void)sprintf(log_buffer, "Next attempt to reconfirm reservation will be made on %s", str_time);
+					(void)snprintf(log_buffer, sizeof(log_buffer), "Next attempt to reconfirm reservation will be made on %s", str_time);
```

Comment:
You really should use a dynamically allocated buffer for this.  The resv_nodes attribute can get REALLY BIG.

Reply:
there is no resv_nodes in the above snprintf().

Reply:
Oh, I'm sorry.  I thought this was a different place in the code.  You are right.